policyengine_us/parameters/gov/states/la/tax/income/credits/school_readiness/rate.yaml

```
@@ -0,0 +1,36 @@
+description: Louisiana provides this percentage of the cdcc as school readiness credit, based on the quality rating of the child care facility.
```

Spell out acronyms in descriptions. Capitalize acronyms in labels. Distinguish tax programs between US and state-specific, especially when they exist in both systems (like the CDCC).
```suggestion
description: Louisiana matches this fraction of the Louisiana child care credit as the school readiness credit, based on the quality rating of the child care facility.
```